chatkit-core/build.gradle

```diff
@@ -19,9 +19,13 @@ dependencies {
     }
 
     implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
-    implementation "org.reduxkotlin:redux-kotlin:0.3.1"
-    api 'com.squareup.okhttp3:okhttp:3.12.6'
+
```

Comment:
do we need extra lines between all the implementations? Can we be consistent if we do?

Reply:
I used it to group related dependencies, i.e. OKHttp/Gson are used for communication with the backend.